gobblin-utility/src/main/java/gobblin/util/ParallelRunner.java

```diff
@@ -205,6 +198,37 @@ public Void call()
     }));
   }
 
+  /**
+   * Delete a path.
+   * @param path path to be deleted.
```

Comment:
Please add `* <p>
    *   This method submits a task to serialize the {@link State} object and returns immediately
    *   after the task is submitted.
    * </p>`.